VERSION

```diff
@@ -1 +1 @@
-6.0.1
+6.1.0
```

Comment:
No breaking change, because all the interfaces remain the same, correct?

Reply:
That is the idea, correct. But not sure how we can validate this to 100%.